### PR TITLE
CMOS-271: Allow overriding metrics port when adding cluster

### PR DESCRIPTION
**Pull request by markspolakovs**
People may have their Prometheus exporters on ports other than 9091. Give them an option to override it, while still using the management port for 7.0 clusters.

### CODE REVIEW

**Review by charlie-hayes — COMMENTED** [review]
Looks good to me, I'll let one more person have a look before merging.

**Review by redm2 — APPROVED** [review]

**Pull request merged**
